docs/orchestrating-elastic-stack-applications/enterprise-search.asciidoc

```diff
@@ -0,0 +1,242 @@
+:page_id: enterprise-search
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
```

Comment:
This is how we do it everywhere today, but I wonder if there's a way to embed the branch name here too so that if someone is browsing the 1.1 branch it will link them to that docs page